src/app/loadout-builder/mod-utils.ts

```diff
@@ -55,13 +65,33 @@ function stringifyMods(permutation: (PluggableInventoryItemDefinition | null)[])
 }
 
 interface ItemEnergy {
+  /** The energy currently used by slot dependant mods. */
```

Comment:
So we exclude general mods from this?

Reply:
Yeah general mods are slot independent so the are part of the big mod checking loop. This allows us to pre-calculate the energy cost of slot dependent mods like arms or class item mods.